llumnix/llumlet/local_migration_scheduler.py

```diff
@@ -11,23 +11,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
-from llumnix.llumlet.migrating_request import MigratingRequest
+from typing import List, Optional
+import numpy as np
+
+from llumnix.llumlet.request import LlumnixRequest, RequestInferenceType
 from llumnix.backends.backend_interface import BackendInterface
 
 class LocalMigrationScheduler:
     def __init__(self, request_migration_policy: str, backend_engine: BackendInterface) -> None:
         self.request_migration_policy = request_migration_policy
         self.backend_engine = backend_engine
 
-    def get_migrate_out_request(self) -> Optional[MigratingRequest]:
-        # TODO(s5u13b): remove the if-else codes
-        migrate_out_request: MigratingRequest = None
+    def get_migrate_out_request(self, min_request_len=0, max_request_len=np.inf) -> Optional[LlumnixRequest]:
```

Comment:
do we give up migrating prefill type requests?

Reply:
yes, we don't migrate request in prefill stage